neptunecontrib/monitoring/xgboost_monitor.py

```diff
@@ -0,0 +1,219 @@
+#
+# Copyright (c) 2020, Neptune Labs Sp. z o.o.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+import os
+import tempfile
+
+import neptune
+import xgboost as xgb
+
+
+def neptune_callback(log_model=True,
```

Comment:
I love this integration.

Sending feat importance and trees is so cool!

Reply:
Thanks!